lib/async_hooks.js

```diff
@@ -215,4 +315,5 @@ module.exports = {
   executionAsyncResource,
   // Embedder API
   AsyncResource,
+  AsyncLocalStorage,
```

Comment:
I don't know what "Embedder API" means here, but I think `AsyncLocalStorage` is public API and can be the first in the exports (alphabetical order).

Reply:
Good catch. Done!